src/components/PeoplePicker/PeoplePicker.scss

```diff
@@ -116,7 +131,7 @@ $personaItemHeight: 42px;
   border: 1px solid $ms-color-neutralTertiaryAlt;
   display: none;
   margin-bottom: -1px;
-  max-width: 340px;
+  min-width: 340px;
```

Comment:
@Jahnp, @atneik is setting the width to width: 100% an acceptable solution then?